docs/release-notes.txt

```diff
@@ -8,6 +8,128 @@
  Version 0.13
 **************
 
+Version 0.13.8
+==============
+
+**Release Date:** November 17, 2020
+
+**New Features**
+
+-  Deployment: When using ``det-deploy aws``, you can now specify the 
+   subnet that the agents should be launched into (if you are using the 
+   simple deployment type). This allows users to explicitly pick a
+   subnet/availability zone that has GPU instances.
+
+-  Security: Allow the agent to use different addresses for master 
```

Comment:
I don't really follow this or understand why it is useful.

Reply:
There's an expanded version coming from the release spreadsheet:

```
Allow the master to use a TLS certificate that is valid for a different name than the agents use to connect to it. This ability is useful in situations where the master is accessed using multiple different addresses (e.g., private and public IP addresses of a cloud instance).

The agent now accepts a ``--security-tls-master-cert-name`` option to override the expected name in the master's TLS certificate. The CLI uses the ``DET_MASTER_CERT_NAME`` environment variable for the same purpose.
```